server/fleet/agent_options.go

```diff
@@ -211,35 +211,27 @@ type osqueryAgentOptions struct {
 
 // NOTE: generate automatically with `go run ./tools/osquery-agent-options/main.go`
 type osqueryOptions struct {
-	AuditAllowAcceptSocketEvents        bool   `json:"audit_allow_accept_socket_events"`
```

Comment:
So these that are being moved to the other struct were incorrectly put here (meaning these require a restart to take effect)?

Reply:
These are linux-only options, so I moved them to the linux struct. In practice, this should make no difference for users, since we don't actually validate options based on OS of the host.